### ISSUE TITLE
Dropdown arrow not visible with dark backgrounds

### ISSUE THREAD

**Issue by edwardhorsford** [alphagov/accessible-autocomplete#227]
If a user uses custom css and a dark background colour, the dropdown arrow is not visible. I think this is because the arrow continues to be black.

Example with black background:
![screen shot 2017-09-06 at 13 18 34](https://user-images.githubusercontent.com/2204224/30111552-02241c22-9306-11e7-980b-519bcf45afed.png)

Example with light background:
![screen shot 2017-09-06 at 13 18 06](https://user-images.githubusercontent.com/2204224/30111562-0a1a26ec-9306-11e7-9657-35d3d06fa475.png)

**Comment by nickcolley** [comment]
We may be able to add `fill: currentColour` to ensure the SVG arrow is the same colour as the border.

**Comment by edwardhorsford** [comment]
Thanks @nickcolley that seems to work.

Light background:
![screen shot 2017-09-06 at 13 32 32](https://user-images.githubusercontent.com/2204224/30112084-fb662bd0-9307-11e7-94a6-79e8ed3eef78.png)

Dark background:
![screen shot 2017-09-06 at 13 32 47](https://user-images.githubusercontent.com/2204224/30112092-ff353c88-9307-11e7-805c-1ecbe8a8f3f1.png)

**Comment by nickcolley** [comment]
So not so easy after all.

At the moment, the autocomplete relies on a transparent background with the icon underneath the input.

This makes sure that the icon can not obstruct clicking the input.

When overriding colours, the background will be set to 'white'. Which hides the icon.

If we move this icon above the input, it now will obstruct opening the input if clicked.

We can add `pointer-events: none` to this element, which prevents any clicks from registering on the icon which solves the problem but this is only available in IE11+.

https://caniuse.com/#feat=pointer-events, any older browsers would still have this obstruction.

This leaves a few options:

1. Bump supported browsers from IE9 to IE11, and only enhance when in IE11
2. Add click handlers to the icon, this feels brittle but is an option.
3. Use [@supports](https://developer.mozilla.org/en-US/docs/Web/CSS/%40supports) and only put the icon on top if pointer-events is supported.

**Comment by andymantell** [comment]
Just to add some more information on this - this bug is rearing it's head when attempting to use the accessible-autocomplete under the new govuk design system, which happens to set background-color to white on the body element - meaning that the arrow is no longer visible.

**Comment by 36degrees** [comment]
> Just to add some more information on this - this bug is rearing it's head when attempting to use the accessible-autocomplete under the new govuk design system, which happens to set background-color to white on the body element - meaning that the arrow is no longer visible.

Eurgh. I _think_ that's actually a separate (but related) issue to this – the arrow SVG has a `z-index` of `-1` which I think is putting it _under_ the body element, so if you have any background colour set at all then it disappears.

Would you mind raising a separate issue?

**Comment by andymantell** [comment]
Yes sorry it is not the same issue. I got fixated on the fact that the solution would be the same for both problems. I'll raise a new ticket...
